syntax = "proto3";
package info;
message Req {
    string filename = 1;
}
message Info {
    string bankno = 1;
    float money = 2;
    string bank = 3;
    string name = 4;
    string remark = 5;
    string time = 6;
}
message ArrInfo {
    repeated Info Ainfo = 1;
}
message Rsp{
    repeated ArrInfo villagesInfo = 1; //repeated villiage info
    ArrInfo otherInfo = 2;
    bool errCode = 3;
}
service GetInfoService {
    rpc GetInfo (Req) returns (Rsp);
}

//上面的是导入银行表提交给grpc的数据

